src/RpcServer/RpcServer.Blockchain.cs

```diff
@@ -258,10 +251,10 @@ protected virtual JToken FindStorage(JArray _params)
         [RpcMethod]
         protected virtual JToken GetTransactionHeight(JArray _params)
         {
-            UInt256 hash = UInt256.Parse(_params[0].AsString());
+            UInt256 hash = Result.Ok_Or(() => UInt256.Parse(_params[0].AsString()), RpcError.InvalidParams.WithData($"Invalid Transaction Hash: {_params[0]}"));
             uint? height = NativeContract.Ledger.GetTransactionState(system.StoreView, hash)?.BlockIndex;
             if (height.HasValue) return height.Value;
-            throw new RpcException(-100, "Unknown transaction");
+            throw new RpcException(RpcError.UnknownTransaction);
```

Comment:
Fix the https://github.com/neo-project/neo-modules/pull/815/files#r1391403046, please.